eslint.config.js

```diff
@@ -0,0 +1,40 @@
+const globals = require('globals');
+
+
+const styleRules = {
+    quotes: ['error', 'single', { avoidEscape: true }],
+    'no-trailing-spaces': 'error',
+    indent: 'error',
+    'linebreak-style': ['error', 'unix'],
+    semi: ['error', 'always'],
+    'brace-style': ['error', '1tbs', { allowSingleLine: true }],
+    'keyword-spacing': 'error',
+    'space-before-blocks': 'error',
+    'space-before-function-paren': [
+        'error',
+        { anonymous: 'always', named: 'never' },
+    ],
+    'no-mixed-spaces-and-tabs': 'error',
+    'comma-spacing': ['error', { before: false, after: true }],
+    'key-spacing': ['error', { beforeColon: false, afterColon: true }],
+};
+
+
+module.exports = {
+    languageOptions: {
+        ecmaVersion: 2022,
+        globals: {
+            ...globals.node,
+        }
+    },
+    ignores: ['/coverage/**'],
```

Comment:
Actually don't think we need this to be fair, the `coverage/` dir generated by `nyc` shouldn't include any JS files so ESLint shouldn't try to lint it